### PR TITLE
TransactionTests Update: ttRSValue

### PR DESCRIPTION
**Pull request by marioevz**
Contains updated TransactionTests for category ttRSValue.

### CODE REVIEW

File: src/TransactionTestsFiller/ttRSValue/TransactionWithSvaluePrefixed00Filler.json [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-	"expect" : [
-            {
-                "network" : [">=Frontier"],
-                "result" : "valid"
-            }
-        ],
+    "TransactionWithSvaluePrefixed00" :
+    {
+	    "expectException" : 
+        {
```
Comment (winsvega) [inline src/TransactionTestsFiller/ttRSValue/TransactionWithSvaluePrefixed00Filler.json:5]: here is not exactly prefixed. there was no way to make a prefixed by then. add a copy of this test using big int and add 00 prefixes to s value, but keep it in range.

File: src/TransactionTestsFiller/ttRSValue/TransactionWithSvalue1Filler.json [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-    "TransactionWithSvalue1" : {
-	"expect" : [
-            {
-                "network" : [">=Frontier"],
-                "result" : "valid"
-            }
-        ],
+    "TransactionWithSvalue1" :
+    {
+	    "expectException" :
```
Comment (winsvega) [inline src/TransactionTestsFiller/ttRSValue/TransactionWithSvalue1Filler.json:4]: is it a github bug or ident here is different to the one with transaction?

File: src/TransactionTestsFiller/ttRSValue/TransactionWithRvaluePrefixed00Filler.json [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
-	     "expect" : [
-            {
-                "network" : [">=Frontier"],
-                "result" : "valid"
-            }
-        ],
+    "TransactionWithRvaluePrefixed00" :
+    {
+	    "expectException" : 
+        {
```
Comment (winsvega) [inline src/TransactionTestsFiller/ttRSValue/TransactionWithRvaluePrefixed00Filler.json:5]: same here. need a copy of this test with bigint prefixed 00 and in range. 
(out of range overflow crazy bigint scenarious we can add up later)

File: src/TransactionTestsFiller/ttRSValue/RightVRSTestVPrefixedBy0_3Filler.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-            {
-                "network" : [">=Frontier"],
-                "result" : "invalid"
-            }
-        ],
+    "RightVRSTestVPrefixedBy0_3" :
+    {
+	    "expectException" : 
+        {
+            "<=EIP150": "InvalidVRS",
```
Comment (winsvega) [inline src/TransactionTestsFiller/ttRSValue/RightVRSTestVPrefixedBy0_3Filler.json]: oh, I forgot I actully support <= in network translation

Comment (winsvega) [reply src/TransactionTestsFiller/ttRSValue/RightVRSTestVPrefixedBy0_3Filler.json]: here it should reject I think not with invalid chain id but with prefixed 00, as the v value limitations. need to check with the v value cap.

File: src/TransactionTestsFiller/ttRSValue/TransactionWithSvaluePrefixed00Filler.json (hunk 1 above)
Comment (marioevz) [reply src/TransactionTestsFiller/ttRSValue/TransactionWithSvaluePrefixed00Filler.json:5]: I created the new test case and it actually fails with:
rlp: non-canonical integer (leading zero bytes) for *big.Int, decoding into (types.Transaction)(types.LegacyTx).S
Which I think is ok.

File: src/TransactionTestsFiller/ttRSValue/TransactionWithRvaluePrefixed00Filler.json (hunk 3 above)
Comment (marioevz) [reply src/TransactionTestsFiller/ttRSValue/TransactionWithRvaluePrefixed00Filler.json:5]: Added test case and it correctly fails like the S case.

File: src/TransactionTestsFiller/ttRSValue/TransactionWithSvalue1Filler.json (hunk 2 above)
Comment (marioevz) [reply src/TransactionTestsFiller/ttRSValue/TransactionWithSvalue1Filler.json:4]: The indent was incorrect in the original file, I tried to make indentation consistent in all files.

File: src/TransactionTestsFiller/ttRSValue/RightVRSTestVPrefixedBy0_3Filler.json (hunk 4 above)
Comment (marioevz) [reply src/TransactionTestsFiller/ttRSValue/RightVRSTestVPrefixedBy0_3Filler.json]: Changed the expected to LeadingZerosV, and changed the V value to bigint.
Let me know if this is ok.

File: src/TransactionTestsFiller/ttRSValue/TransactionWithSvalue1Filler.json (hunk 2 above)
Comment (winsvega) [reply src/TransactionTestsFiller/ttRSValue/TransactionWithSvalue1Filler.json:4]: I think you added TAB

File: src/TransactionTestsFiller/ttRSValue/TransactionWithRvalueHighFiller.json [hunk 5]
```diff
@@ -14,9 +13,9 @@
             "nonce" : "0",
             "to" : "095e7baea6a6c7c4c2dfeb977efac326af552d87",
             "value" : "11",
-            "v" : "27",
+            "v" : "28",
```
Comment (winsvega) [inline src/TransactionTestsFiller/ttRSValue/TransactionWithRvalueHighFiller.json:16]: why change?

**Review by winsvega — APPROVED** [review]
looks good

**Pull request merged**